docs/src/model_setup/buoyancy_and_equation_of_state.md

```diff
@@ -20,7 +20,7 @@ end
 ```jldoctest buoyancy
 julia> grid = RectilinearGrid(size=(64, 64, 64), extent=(1, 1, 1));
 
-julia> model = NonhydrostaticModel(grid=grid, buoyancy=nothing)
+julia> model = NonhydrostaticModel(grid=grid, buoyancy=nothing, tracers=(:T, :S))
```

Comment:
This docs section has to be rewritten to reflect that now "no buoyancy" is the default.